code/src/App.js

```diff
@@ -1,9 +1,159 @@
-import React from 'react'
+import * as React from 'react'
+import { useState } from 'react'
+import Avatar from '@material-ui/core/Avatar'
+import AppBar from '@material-ui/core/AppBar'
+import Toolbar from '@material-ui/core/Toolbar'
+import Button from '@material-ui/core/Button'
+import Typography from '@material-ui/core/Typography'
+import { makeStyles } from '@material-ui/core/styles'
+import Container from '@material-ui/core/Container'
+import Stepper from '@material-ui/core/Stepper'
+import Step from '@material-ui/core/Step'
+import StepLabel from '@material-ui/core/StepLabel'
+import { ButtonGroup, Card, CardContent } from '@material-ui/core'
+
+import { Step1 } from './Step1'
+import { Step2 } from './Step2'
+import { Step3 } from './Step3'
+
+export const useStyles = makeStyles((theme) => ({
+  palette: {
+    type: 'dark',
+  },
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main,
+  },
+  form: {
+    width: '100%', // Fix IE11 issue.
+    marginTop: theme.spacing(1),
+  },
+  formControl: {
+    alignItems: 'center',
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}))
 
 export const App = () => {
+  const [name, setName] = useState('')
+  const [favourite, setFavourite] = useState('')
+  const [knowledge, setKnowledge] = useState('')
+  const [section, setSection] = useState(1)
+
+  const handleName = (name) => {
+    setName(name)
+  };
```

Comment:
(I think I commented on the wrong version of this file, sorry if you received that), this `;` should go